hphp/hack/src/heap/hh_shared.c

```diff
@@ -142,11 +142,11 @@
 #if !defined __APPLE__ && !defined _WIN32
   // Linux version for the architecture must support syscall memfd_create
```

Comment:
Yes. Because `SYS_memfd_create` is defined as: `#define SYS_memfd_create __NR_memfd_create`. Originally i sent a pull request that fixes a error due a hardcoded (x86_64) value: changes `return syscall(319, name, flags);` to `return syscall(__NR_memfd_create, name, flags);`. But the reviewer says that we should not rely on include files: "If we rely on the header, then if the header does NOT define __NR_memfd_create, then the binary will never use memfd_create, even if run on a machine whose kernel supports memfd_create. We need runtime checks for memfd_create support, not compile-time checks."